src/ui/Combobox/ComboboxInput.ts

```diff
@@ -73,10 +73,17 @@ export class ComboboxInput {
 
   private handleFocusOut(event: FocusEvent) {
     const newTarget = event.relatedTarget as HTMLElement;
-    const isFocusedOnInput = this.combobox.element.contains(newTarget);
-    if (isFocusedOnInput) {
+    const focusInsideCombobox = this.combobox.element.contains(newTarget);
+    if (focusInsideCombobox) {
       return;
     }
+
+    const comboboxValuesHovered = $$(this.combobox.element).find('.coveo-combobox-values:hover');
```

Comment:
On IE11, the scrollbar mousedown will trigger the focusout/blur of the input, this is how we prevent it

Reply:
Btw, didn't find a way to test the `:hover` for jasmine, I think it will take a manual/integration test